tests/integration/dhcp_test.py

```diff
@@ -114,8 +118,37 @@ def dhcp_env():
 
 @pytest.fixture
 def dhcpcli_up(dhcp_env):
```

Comment:
Please create new fixture like `dhcpcli_up_auto` and place the sleep(5) in it.

Reply:
The current fixture is inconsistent between ipv4 and ipv6. For ipv4 it is set to auto but for ipv6 is somehow does not work. I think we need to be explicit here to avoid these inconsistencies now and in the future.

I will split this change so it will be clear.

Reply:
Done. One commit for changing the way we start dhcpcli interface and another for the test with the addition of the delay.